roles/container-engine/containerd/defaults/main.yml

```diff
@@ -25,9 +25,20 @@ containerd_additional_runtimes: []
 #    type: "io.containerd.kata.v2"
 #    engine: ""
 #    root: ""
+# Example for Gvisor Containers as additional runtime with options:
+#  - name: runsc
+#    type: io.containerd.runsc.v1
+#    engine: ""
+#    root: ""
+#    options:
+#      TypeUrl: '"io.containerd.runsc.v1.options"'
+#      ConfigPath: '"{{ containerd_cfg_dir }}/runsc.toml"'
 
 containerd_base_runtime_spec_rlimit_nofile: 65535
 
+# Definition of the platform for the runtime runsc (used by gvisor)
+containerd_runsc_platform: "kvm"
```

Comment:
From the https://github.com/google/gvisor/commit/501a9aaf495c6916d9161b02719b03af6b18ed6d

The kvm is not the default platform of gvisor. Should the default value do not be  changed :-)